addon/record-array.js

```diff
@@ -33,7 +37,9 @@ if (CUSTOM_MODEL_CLASS) {
     init() {
       super.init(...arguments);
       this._references = [];
-      this._objects = A();
+      if (!this._objects) {
```

Comment:
can we obscure this property even more to prevent access? would be nice to kill off these _ props by moving them to a weakmap or some such

Reply:
Given the _ and the non similarity to public apis, I don't think it's as bad as `content`. It's used in child classes, so having it on the array is nice.